src/components/Input/input.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Input from './index'
+
+export default {
+  title: 'Input',
+}
+
+export const CustomInput = () => (
```

Comment:
In storybook you export multiple named imports, which are components and you customise them as you would like, writing them as a header.
please separate the stories/components.

Reply:
I'm not expert in storybook and I don't like it

Reply:
you can make I suggestion or explain more because I see spending time on other issues will be better

Reply:
https://github.com/yalla-coop/website-redesign/blob/master/src/components/elements/typography/paragraphs.stories.js
Storybook is for showcasing and documenting components

Reply:
there is nothing to hate about it imo